relay-event-normalization/src/normalize/mod.rs

```diff
@@ -990,6 +1061,49 @@ impl<'a> DynamicMeasurementsConfig<'a> {
     }
 }
 
+/// Defines a weighted component for a performance score
+///
+/// Weight is the % of score it can take up (eg. LCP is a max of 35% weight for desktops)
+/// Currently also contains (p10, p50) which are used for log CDF normalization of the weight score
+#[derive(Debug, Default, Clone, Serialize, Deserialize)]
+pub struct PerformanceScoreWeightedComponent {
+    /// Measurement (eg. measurements.lcp) to be matched against. If this measurement is missing the entire
+    /// profile will be discarded.
+    pub measurement: String,
+    /// Weight [0,1.0] of this component in the performance score
+    pub weight: f64,
+    /// p10 used to define the log-normal for calculation
+    pub p10: f64,
+    /// Median used to define the log-normal for calculation
+    pub p50: f64,
+}
+
+/// Defines a profile for performance score
+///
+/// A profile contains weights for a score of 100% and match against an event using a condition.
+/// eg. Desktop vs. Mobile(web) profiles for better web vital score calculation.
+#[derive(Debug, Default, Clone, Serialize, Deserialize)]
+pub struct PerformanceScoreProfile {
```

Comment:
`score_components` -> `scoreComponents`.

```suggestion
#[serde(rename_all = "camelCase")]
pub struct PerformanceScoreProfile {
```